system_tests/common_test.go

```diff
@@ -239,6 +241,11 @@ func createTestL1BlockChainWithConfig(t *testing.T, l1info info, stackConfig *no
 		return nil
 	}})
 
+	stack.RegisterAPIs([]rpc.API{{
```

Comment:
why is this needed? what's wrong with the filterAPI registered by the backend?

Reply:
This is for L1